_includes/manuals/1.14/4.3.6_smartproxy_puppet.md

```diff
@@ -21,7 +21,7 @@ Replace `4.5.0` with the version of Puppet installed on the Puppet master, this
 
 The proxy generates a list of all Puppet classes and their parameters of the Puppet modules inside the environments declared in puppet.conf. These are imported by Foreman to generate the list of classes, smart class parameters and environments that they belong to. The mechanism used depends on the version of Puppet (specified by `:puppet_version`):
 
-* On Puppet 4, the smart proxy will use the environments and resource_type Puppet Server APIs to list classes and parameters.
+* On Puppet 4, the smart proxy will use the environments and environment_classes Puppet Server APIs to list classes and parameters. If you're running Puppet Master or Puppet Server older than 4.4, the API is not available. To use resource_types API, set `:puppet_version` to 4.3 even if you have newer Puppet Master.
```

Comment:
environment_classes is only available on puppet server, while resource_types is also on puppet master (afaiu the difference) which is why I tried to put the info there, anyway I simplified it as suggested, please take another look